### PR TITLE
chore(test): skip tpch q4 in cluster mode

### PR DESCRIPTION
**Pull request by xudong963**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Need time to solve the flakey test, let's skip it temporarily.

Fixes #issue

### CODE REVIEW

File: scripts/ci/ci-run-stateless-tests-cluster.sh [hunk 1]
```diff
@@ -9,4 +9,4 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" >/dev/null 2>&1 && pwd)"
 cd "$SCRIPT_PATH/../../tests" || exit
 
 echo "Starting databend-test"
-./databend-test --mode 'cluster' --run-dir 0_stateless
+./databend-test --mode 'cluster' --run-dir 0_stateless --skip '13_0004_q4'
```
Comment (Xuanwo) [inline scripts/ci/ci-run-stateless-tests-cluster.sh:12]: Please add a comment which links to an exist issue here.

Comment (Xuanwo) [reply scripts/ci/ci-run-stateless-tests-cluster.sh:12]: Oh, we already have one: https://github.com/datafuselabs/databend/issues/8107

Comment (Xuanwo) [inline scripts/ci/ci-run-stateless-tests-cluster.sh:12]:
```suggestion
# The skip should be removed after https://github.com/datafuselabs/databend/issues/8107 is addressed.
./databend-test --mode 'cluster' --run-dir 0_stateless --skip '13_0004_q4'
```

**Review by Xuanwo — APPROVED** [review]

File: scripts/ci/ci-run-stateless-tests-cluster.sh (hunk 1 above)
Comment (xudong963) [reply scripts/ci/ci-run-stateless-tests-cluster.sh:13]: 👍

**Pull request merged**